docs/resources.md

```diff
@@ -246,20 +249,36 @@ where “image-attestor” is the project for creating AttestationAuthority Note
 
 In order to create notes, the service account `gac-ca-admin` must have `containeranalysis.notes.attacher role` on this project.
 
-The Kubernetes secret `foo` must have data fields `private` and `public` which contain the gpg private and public key respectively.
+`publicKeys` is a list of public keys for the AttestationAuthority. Key rotation is supported by listing multiple keys.
+An image is attested if it has an attestation verifiable by ANY of the public keys.
 
 
-`publicKeys` is a list of public keys for the Kubernetes secret. Key rotation is supported by listing multiple keys.
-An image is attested if it has an attestation verifiable by ANY of the public keys.
+Each public key contains an ID. Signatures verified by BinAuthz must include the ID of the public key that can be used to 
+verify them, and that ID must match the contents of this field exactly. Additional restrictions on this field can be 
+imposed based on which public key type is encapsulated. See the documentation on publicKey cases below for details.
+
+There are two types of public keys: PGP keys and PKIX keys. These keys are defined like Binauthz 
+[AttestorPublicKeys](https://cloud.google.com/binary-authorization/docs/reference/rest/v1/projects.attestors#attestorpublickey), with
+an additional `keyType` field.
+
+### PGP Public Keys
+
+| Field    | Type    | Value   |
+|----------|----------|---------------|
+| `keyType`  | string   | "PGP"| 
+| `keyId`  | string   | Optional: either OpenPGP RFC4880 V4 fingerprint of the key payload or blank. If left blank, the `keyId` will be computed as the key's OpenPGP fingerprint.| 
+| `asciiArmoredPgpPublicKey`  | string | A base64-encoded ASCII-armored representation of a PGP public key, as the entire output by the command `gpg --export --armor foo@example.com | base64` (either LF or CRLF line endings). |
+| `pkixPublicKey` | PkixPublicKey object | empty|
 
+### PKIX Public Keys
```

Comment:
Optional: Perhaps it's enough to only say that this is not implemented yet.  Your call...

Reply:
Done.